test/e2e/specs/editor-tracking/editor-tracking__fse-template-events.ts

```diff
@@ -79,7 +79,7 @@ describe(
 
 		// These events are currently broken :(. See https://github.com/Automattic/wp-calypso/issues/62924.
 		// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
**todo**: remove this line prior to merge as the eslint-disable is no longer necessary.

Reply:
Good catch @worldomonation. That's been done.